tests/test_json_parser.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from validator.constants import FIREFOX_GUID
+from validator.json_parser import ManifestJsonParser
+
+
+manifest_json = """{
+        "name": "My Awesome Addon",
+        "version": "1.25",
+
+        "applications": {
+            "gecko": {
+                "id": "my@awesome.addon"
+            }
+        }
+    }"""
```

Comment:
On second thought, you're right. We have json strings all over the place. It just felt better to not put large amounts of JSON into the module for sake of clarity.